cmd/ops/unused.go

```diff
@@ -0,0 +1,87 @@
+package ops
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/sketch-hq/gql-lint/output"
+	"github.com/sketch-hq/gql-lint/unused"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//Flags
+	queryFiles string
+
+	//Command
+	unusedCmd = &cobra.Command{
+		Use:   "unused [flags] queries_directories",
+		Short: "Find unused deprecated fields",
+		Args:  unusedCmdArgsValidation,
+		RunE:  unusedCmdRun,
+	}
+)
+
+func init() {
+	Program.AddCommand(unusedCmd)
+	unusedCmd.Flags().StringVar(&schemaFile, schemaFileFlag, "", "Server's schema file (required)")
+	unusedCmd.MarkFlagRequired(schemaFileFlag) //nolint:errcheck // will err if flag doesn't exist
+}
+
+func unusedCmdArgsValidation(cmd *cobra.Command, args []string) error {
```

Comment:
I will look into how we can avoid repeating code in the commands, but is this good as it is now

Reply:
yeah I figured once we settle these, I should go and unify these things